api/post.go

```diff
@@ -680,10 +687,20 @@ func sendNotifications(c *Context, post *model.Post, team *model.Team, channel *
 								msg.DeviceId = strings.TrimPrefix(session.DeviceId, model.PUSH_NOTIFY_ANDROID+":")
 							}
 
-							if channel.Type == model.CHANNEL_DIRECT {
-								msg.Message = senderName + userLocale("api.post.send_notifications_and_forget.push_message")
+							if *utils.Cfg.EmailSettings.PushNotificationContents == model.FULL_NOTIFICATION && model.IsValidHttpsUrl(*utils.Cfg.EmailSettings.PushNotificationServer) {
```

Comment:
The ticket says: "This option is only available if the Push Notification Server is using an "https" protocol to encrypt push notifications sent by the server." so that's why, nonetheless if its not using SSL then it will send the generic notifications